library/core/src/cmp.rs

```diff
@@ -810,9 +827,12 @@ pub trait Ord: Eq + PartialOrd<Self> {
     /// ```
     #[must_use]
     #[stable(feature = "clamp", since = "1.50.0")]
+    #[default_method_body_is_const]
     fn clamp(self, min: Self, max: Self) -> Self
     where
         Self: Sized,
+        Self: ~const Drop,
+        Self: ~const PartialOrd,
```

Comment:
This is a new bound that wasn't here before. Ideally we'd have a `~const PartialOrd<Self>` parent bound on the trait itself. That only makes sense with the proposed `#[const_trait]`, so I guess this is blocked on that?

Reply:
Yep